plugins/source/aws/resources/services/emr/clusters.go

```diff
@@ -37,7 +37,7 @@ func Clusters() *schema.Table {
 				Resolver: client.ResolveTags,
 			},
 		},
-		Relations: []*schema.Table{clusterInstanceFleets(), clusterInstanceGroups(), clusterInstances()},
+		Relations: []*schema.Table{clusterInstanceFleets(), clusterInstanceGroups(), clusterInstances(), notebookExecutions(), steps()},
```

Comment:
I would update this to be one table() per line so that further modifications are easily visible and mergable